### PR TITLE
[master][KOGITO-1703] Wrong build resource description in KogitoApp CRD

### PR DESCRIPTION
**Pull request by sutaakar**
https://issues.redhat.com/browse/KOGITO-1703

Many thanks for submiting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors guide](CONTRIBUTING.MD#sending-a-pull-request)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a unit test that verifies it
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

File: deploy/crds/app.kiegroup.org_kogitoapps_crd.yaml [hunk 1]
```diff
@@ -206,8 +206,7 @@ spec:
                     more information, see https://www.graalvm.org/docs/reference-manual/aot-compilation/.
                   type: boolean
                 resources:
-                  description: Resources for build pods. Default limits are 1GB RAM/0.5
-                    CPU on JVM and 4GB RAM/1 CPU for native builds.
+                  description: Resources for S2I build pods.
```
Comment (spolti) [inline deploy/crds/app.kiegroup.org_kogitoapps_crd.yaml:209]: builder pods?

Comment (ricardozanini) [reply deploy/crds/app.kiegroup.org_kogitoapps_crd.yaml:209]: Please do not change the CRDs directly!!! They are auto-generated and updated with the `operator-sdk` command line. 

Please update the `_types.go` file and then run `make vet` to apply the changes.

**Comment by sutaakar** [comment]
Change applied to the kogitoapp_types.go

**Comment by ricardozanini** [comment]
This one should be on 0.9.x as well? Could you please update the JIRA `fixVersion` with the target versions? Many thanks!

**Comment by sutaakar** [comment]
@ricardozanini Done, will open another PR for 0.9.x

**Pull request merged**